src/linker/Linker/Driver.cs

```diff
@@ -789,14 +790,18 @@ public int Run (ILogger? customLogger = null, bool throwOnFatalLinkerException =
 				if (throwOnFatalLinkerException)
 					throw;
 				return lex.MessageContainer.Code ?? 1;
+			}
+			catch (ArgumentException) {
```

Comment:
What is special on ArgumentException ?

Reply:
This was thrown when a custom step tries to use an invalid error code. It's no longer relevant to the PR though.